wallaby.js

```diff
@@ -0,0 +1,89 @@
+const babelConfig = require("./babel.config.js");
+
+module.exports = function(wallaby) {
+    const tests = [
+        // "test/*-spec.js",
```

Comment:
I disabled running these tests b/c I hadn't created `.prettierrc` and it was reformatting the file.  It should be safe to re-enable this now.